src/Sarif/Sarif.csproj

```diff
@@ -45,9 +45,8 @@
     <Content Include="Schemata\sarif-$(SchemaVersionAsPublishedToSchemaStoreOrg).json" />
     <Content Include="Schemata\sarif-external-property-file-$(SchemaVersionAsPublishedToSchemaStoreOrg).json" />
 
-    <!-- Link=false: Don't display this item in Solution Explorer. -->
     <!-- This item will be added to the NuGet package, however. -->
-    <Content Include="..\ReleaseHistory.md" Link="false" />
```

Comment:
>false [](start = 50, length = 5)

The effect of this was that when you edited ReleaseHistory.md, the tab in the VS editor showed the file name "false".

Reply:
oh wow, thanks for fixing this!

---
In reply to: [299229228](https://github.com/microsoft/sarif-sdk/pull/1558#discussion_r299229228) [](ancestors = 299229228)